PSKoans/Koans/Cmdlets 1/AboutPSProviders.Koans.ps1

```diff
@@ -127,17 +127,19 @@ Describe 'Environment Provider' {
 }
 
 Describe 'FileSystem Provider' {
-    $Path = 'TestDrive:' | Join-Path -ChildPath 'File001.tmp'
+    #Get-TempDirectory is a private function in PSKoans
+    #This should return a path to your temp directory.
+    $Path = Get-TempDirectory
     if (-not (Test-Path $Path)) {
-        New-Item -Path $Path > $null
-
-        -join (1..10000 | ForEach-Object {
-                Get-Random -Minimum 32 -Maximum 96 |
-                    ForEach-Object {[char]$_ -as [string]}
-                if ((Get-Random) -match '[25]0$') {
-                    "`n"
-                }
-            }) | Set-Content -Path $Path
+
+        $FileContent = @'
+PSKOANS!
+The Env: drive contains system environment data. Its contents can vary wildly from OS to OS,
+especially between Windows, Mac, and Linux, for example.
+
+The only shared Env: items across all OSs currently are Path and PSModulePath.
+'@
+        Set-Content -Path $Path -Value $FileContent
```

Comment:
This section is for the FileSystem provider, though. The text itself might seem a little out of place in context, I'm thinking?